tests/export_test.py

```diff
@@ -57,19 +58,36 @@ def tearDownModule():
   prev_xla_flags()
 
 ### Setup for testing lowering with effects
-class TestingOrderedEffect1(effects.Effect):
-  __str__ = lambda _: "TestingOrderedEffect1"
+@dataclasses.dataclass(frozen=True)
+class ForTestingOrderedEffect1(effects.Effect):
+  __str__ = lambda _: "ForTestingOrderedEffect1"
```

Comment:
FYI `dataclass()` will generate a very similar `__str__` for you.

Reply:
Done